src/scss/index.scss

```diff
@@ -1 +1,22 @@
+@import "variables";
+@import "./header";
+@import "./login";
+@import "./feed";
+@import "./profile";
+@import "../../node_modules/bootstrap/scss/bootstrap";
 
+$small-screen: 768px;
```

Comment:
You don't need these. Bootstrap handles breakpoints well

Reply:
Fixed